src/windows/find_tools.rs

```diff
@@ -482,28 +493,42 @@ mod impl_ {
     ) -> Option<(PathBuf, PathBuf, PathBuf, PathBuf, Option<PathBuf>, PathBuf)> {
         let version = vs15plus_vc_read_version(instance_path)?;
 
-        let host = match host_arch() {
-            X86 => "X86",
-            X86_64 => "X64",
-            // There is no natively hosted compiler on ARM64.
-            // Instead, use the x86 toolchain under emulation (there is no x64 emulation).
-            AARCH64 => "X86",
+        let hosts = match host_arch() {
+            X86 => &["X86"],
+            X86_64 => &["X64"],
```

Comment:
Question for reviewers: now that we have a "fallback" mechanism in place, should x64 fallback to x86?

Reply:
IMO think it's ok to have it, though given that x64 is widely available it's unlikely to be used.